code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -447,7 +447,7 @@
 	if(!linked_extract || !linked_extract.loc) //Sanity checking
 		qdel(src)
 		return
-	if(linked_extract && linked_extract.loc != owner && linked_extract.loc.loc != owner)
+	if(linked_extract.loc != owner && linked_extract.loc.loc != owner)
```

Comment:
```suggestion
	if(linked_extract?.loc != owner && linked_extract.loc.loc != owner)
```
This implies that you think the linked extract cannot be null? am i seeing this correct if you add a ? before the var access it will check for the object beeing null